plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -625,6 +679,26 @@ global proc int mayaUsdTranslatorExport (string $parent,
 //        0 otherwise.
 //
 {
+    // check for Export All or Export Selection
+    // check for export option box window first, if it doesn't exist means user opened the main export window

```

Comment:
Check the global variable from Maya to tell if we are in ExportAll or ExportSelection